core/services/ocr2.go

```diff
@@ -159,6 +179,9 @@ func (o *OCR2) Start() {
 	// start contract subscription
 	go o.contract.Start()
 
+	// start client for the monitoring endpoint which pushes telemetry to OTI
+	go o.monitoringEndpoint.Start()
```

Comment:
Then here we don't start a new connection, as this will create N connections for N OCR jobs.

We want one durable connection between the Relay & remote OTI service. Then every OCR job (service that runs the job) uses that one connection to report its telemetry.

So OCR service should just require a TelemetryClient as a dependency. As this is a dependency, not a new client we construct ourselves we expect it's already connected by some other system service (telemetry/monitoring).

Then the OCR service directly uses the telemetry port (client) to do things (send telemetry).